app/src/main/java/invoice_automation/module/QuickBooksModule.java

```diff
@@ -84,28 +84,28 @@ public List<Customer> getAllCustomers() {
      * @return A copy of the Customer updated or created
      */
     public Customer updateCustomerFromSchool(@NonNull School school) {
-        boolean addNew = true;
         // Get customer associated with the school, if it exists
         Customer schoolCustomer = new Customer();
```

Comment:
nit: don't need a whole customer, just have two variables for id and sync token (less memory used this way)